pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -669,6 +669,19 @@ func (c *ConfigMapUnpacker) ensureJob(cmRef *corev1.ObjectReference, bundlePath
 	if err != nil {
 		return
 	}
+
+	// This is to ensure that any unpack job that was successful but did not contain the label was taken into account.
```

Comment:
nitpick, since existing jobs could also potentially be failed:
```suggestion
	// This is to ensure that we account for any existing unpack jobs that may be missing the label
```